__manifest__.py

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Open Academy",
+
+    "summary": """This is a summary of Open Academy Module.""",
+
+    "description": """This is a description of Open Academy Module.""",
+
+    "author": "Vauxoo",
+
+    "website": "https://www.vauxoo.com",
+
+    "category": "Uncategorized",
+
+    "version": "15.0.1.0.0",
+
+    "license": "LGPL-3",
+
+    "depends": ["base"],
+
+    "data": [
+        "views/openacademy_menu.xml",
+        "views/course_view_form.xml",
```

Comment:
Views should be named and grouped by model name, e.g. `openacademy_course_view.xml`. For more info, check [the guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming)